pkg/ctl/implementation.go

```diff
@@ -47,21 +47,22 @@ const (
 		"`vexctl generate` when generating VEX data for a release or\n" +
 		"a specific artifact.\n\n" +
 		"To add new statements to publish data about a vulnerability,\n" +
-		"download [vexctl](https://github.com/openvex/vexctl)\n" +
-		"and append new statements using `vexctl add`. For example:\n\n" +
+		"download [vexctl] and append new statements using\n" +
```

Comment:
ah sorry

i think you can keep the link

```suggestion
		"download [vexctl](https://github.com/openvex/vexctl) and append new statements using\n" +
```

Reply:
The auto generated template will be like this, because it uses the [shortcut reference link](https://github.github.com/gfm/#shortcut-reference-link).

---
# OpenVEX Templates Directory

This directory contains the OpenVEX data for this repository.
The files stored in this directory are used as templates by
`vexctl generate` when generating VEX data for a release or
a specific artifact.

To add new statements to publish data about a vulnerability,
download [vexctl] and append new statements using
`vexctl add`. For example:

```
vexctl add --in-place main.openvex.json --product pkg:oci/test --vuln CVE-2014-1234567 --status under_investigation
```

That will add a new VEX statement expressing that the impact of
CVE-2014-1234567 is under investigation in the test image. When
cutting a new release, for `pkg:oci/test` the new file can be
incorporated to the release's VEX data.

## Read more about OpenVEX

To know more about generating, publishing and using VEX data
in your project, please check out the [vexctl repository and
documentation][vexctl].

OpenVEX also has an [examples repository] with samples and docs.


[vexctl]: https://github.com/openvex/vexctl
[examples repository]: https://github.com/openvex/examples

---

We use the same shortcut in both references to `vexctl`, because the original target link is the same.

![image](https://github.com/user-attachments/assets/1866a274-6204-4894-9238-782b4fd55f97)